roman_numeral/roman_numerals.py

```diff
@@ -0,0 +1,40 @@
+"""Given a roman numeral, find the integer equivalent"""
+
+# roman_numerals = ["I", "II", "III", "IV", "V", "VI", "VII", "VIII", "IV", "X"]
+
+
+number = 6
+
+
+roman_numeral_values = {1: "I", 5: "V", 10: "X"}
+
+
+def multiples_of_five():
+    """Create a list of numbers that are multiples of 5"""
+    multiples = []
+    for nums in range(1, number + 1):
+        if nums % 5 == 0:
+            multiples.append(nums)
+    return multiples
+
+
+def conversion(number):
+    """Convert an Arabic number to its roman numeral equivalent"""
+    if number == 0:
+        return ""
+
+    else:
+        convert = []
+        reduced_num = number
+
+        if number in multiples_of_five():
+            convert.append(roman_numeral_values[number])
+            reduced_num -= number
+        else:
```

Comment:
So your question on the special case of `IV`...

so this is the third type of condition you could have. You are already dealing with the first two correctly. I'm going to leave it to you to figure out how to test for this condition, and then, how to deal with it. But start with the conditional test you'll need.